quickstart-template/webhook/app.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const app = express();
+
+app.use(express.json());
+
+app.post('/callback', (req, res) => {
+    const data = req.body;
+    console.log(JSON.stringify(data, null, 4));
+    const response = {
+        message: 'Data received',
+        received_data: data
+    };
+    res.json(response);
```

Comment:
Can we show examples of how they can use the webhook events?

Reply:
we'll be showing more use cases later